### PR TITLE
8263855: Use the blessed modifier order in java.management/naming

### PR DESCRIPTION
**Pull request by alblue**
As with #2993 changing the order of `final static` to `static final` for the `java.management`, `java.management.rmi` and `java.naming` modules.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8263855](https://bugs.openjdk.java.net/browse/JDK-8263855): Use the blessed modifier order in java.management/naming

### Reviewers
 * [Claes Redestad](https://openjdk.java.net/census#redestad) (@cl4es - **Reviewer**)
 * [Aleksei Efimov](https://openjdk.java.net/census#aefimov) (@AlekseiEfimov - Committer)
 * [Daniel Fuchs](https://openjdk.java.net/census#dfuchs) (@dfuch - **Reviewer**)

### Download
To checkout this PR locally:
`$ git fetch https://git.openjdk.java.net/jdk pull/3078/head:pull/3078`
`$ git checkout pull/3078`

To update a local copy of the PR:
`$ git checkout pull/3078`
`$ git pull https://git.openjdk.java.net/jdk pull/3078/head`

### CODE REVIEW

**Comment by alblue** [comment]
Would someone mind creating a bug for me? In addition, if it would help, we could create a parent bug for applying cleanups  and associate #2993 and #2991.

**Comment by alblue** [comment]
@cl4es would you mind creating a parent task of something like "Source code cleanups" and then another sub task for this change please?

**Comment by cl4es** [comment]
Created the subtask https://bugs.openjdk.java.net/browse/JDK-8263855 for this along with an umbrella RFE.

**Comment by alblue** [comment]
Thanks @cl4es -- do I need to update the git commit message as well, or is updating the title of the PR sufficient? I recall you suggesting not to do amend/rebases previously.

**Comment by cl4es** [comment]
No, the git commit messages here doesn't matter, the bots will use the PR name when merging into master. It's good form to use reasonably consistent commit messages as you add commits to a PR, but altering commit history is not necessary and might even be disruptive once a PR has been opened.

There's the /summary command you could use to add additional comments to the final commit, see https://wiki.openjdk.java.net/display/SKARA/Pull+Request+Commands#PullRequestCommands-/summary

**Comment by alblue** [comment]
Thanks @cl4es will adjust my submissions appropriately in the future.

File: src/java.naming/share/classes/com/sun/jndi/ldap/EventQueue.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
  * @author      Rosanna Lee (modified for JNDI-related events)
  */
 final class EventQueue implements Runnable {
-    final static private boolean debug = false;
+    static final private boolean debug = false;
```
Comment (cl4es) [inline src/java.naming/share/classes/com/sun/jndi/ldap/EventQueue.java:48]: Move private to the front?

File: src/java.naming/share/classes/com/sun/jndi/ldap/EventSupport.java [hunk 2]
```diff
@@ -113,7 +113,7 @@
  * @author Rosanna Lee
  */
 final class EventSupport {
-    final static private boolean debug = false;
+    static final private boolean debug = false;
```
Comment (cl4es) [inline src/java.naming/share/classes/com/sun/jndi/ldap/EventSupport.java:116]: Move private to the front?

File: src/java.naming/share/classes/com/sun/jndi/ldap/LdapSchemaCtx.java [hunk 3]
```diff
@@ -386,7 +386,7 @@ final private void modifyServerSchema(Attributes origAttrs,
         info.modifyAttributes(myEnv, mods);
     }
 
-    final static private class SchemaInfo {
+    static final private class SchemaInfo {
```
Comment (cl4es) [inline src/java.naming/share/classes/com/sun/jndi/ldap/LdapSchemaCtx.java:389]: Move private to the front?

**Review by cl4es — CHANGES_REQUESTED** [review]
Overall good - found a few cases where the private modifier is in the wrong place that ought to be fixed as well.

**Review by cl4es — APPROVED** [review]

**Review by AlekseiEfimov — APPROVED** [review]
`java.naming` module changes look good to me.

**Review by dfuch — APPROVED** [review]

**Pull request closed without merging**